vmdk_plugin/main.go

```diff
@@ -97,8 +98,8 @@ func main() {
 	var driver volume.Driver
 
 	// Define command line options
-	driverName := flag.String("driver", "vmdk", "Volume driver")
-	logLevel := flag.String("log_level", "debug", "Logging Level")
+	driverName := flag.String("driver", vmdkDriver, "Volume driver")
```

Comment:
Its possible to start two listeners on two separate sockets, the sockets are named after the driver - vmdk.sock, vsphere.sock. One instance of the plugin process runs with the two listeners in this case.